ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -62,6 +62,24 @@ tests:
       IMAGE_REPO: notebooks
       RELEASE_VERSION: jupyter-minimal-ubi8-python-3.8
     workflow: opendatahub-io-ci-image-mirror
+- as: notebook-jupyter-data-science-ubi8-python-3-8-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: jupyter-data-science-ubi8-python-3.8
```

Comment:
Where is this image built? You are trying to mirror something that does not exist.